src/libraries/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -681,32 +693,35 @@ public static IEnumerable<object[]> SerializableObjects()
 
-            SqlError sqlError = null;
-            // .NET Framework has no constructor overload for passing an exception
-            if (PlatformDetection.IsNetFramework)
+            if (!PlatformDetection.IsBrowser) // System.Data.SqlClient doesn't work on Browser
```

Comment:
@bartonjs, @joperezr, do we still want to test serialization of SqlClient types now that SqlClient is no longer in dotnet/runtime?  Separate from this PR, seems like we could probably delete this section.

Reply:
I'm fine with removing them given they would effectively not be catching anything any longer.

Reply:
Logged https://github.com/dotnet/runtime/issues/38949 to track this.